cloudify_cli/inputs.py

```diff
@@ -24,7 +24,7 @@
 
 
 # TODO: Add test for inputs as JSON/YAML string
-def inputs_to_dict(resources, dot_hierarchy=False):
+def inputs_to_dict(resources, dot_hierarchy=False, deleting=False):
```

Comment:
Maybe instead of `dot_hierarchy=False, deleting=False` it is better to use `**kwargs`